### ISSUE TITLE
Suggest use the windows10 spring  creator's Fluent API

### ISSUE THREAD

**Issue by wuyang26** [sourcechord/FluentWPF#20]
Can you use the windows10 spring  creator's Fluent API?
Now You using the effect is like the windows7's Aero ,the newest Telerik WPF Controls use it too.
but it's fuzzy effect is not enough strong,the background icon is too clear.
the DevExpress's Winform controls(V18.1) use the window10's fluent API,
the fuzzy effect is very good,it is better than the window10 self APP,because
it's fuzzy effect between the windows7's Aero and window10's Fluent Design.

**Comment by XeZrunner** [comment]
I agree about using the new Acrylic API that has been introduced to Win32 lately.

**Comment by sourcechord** [comment]
Hi, @wuyang26 , @XeZrunner.
Sorry for replying so late.

I recognized this issue, and I've tried to support the new Acrylic API.
But, the new API couldn't perform well in Windows10(1803) with WPF's WindowChrome.

By the way, I tried the new Acrylic API with Windows10(1809) a few days ago.
Their is no problem. More fluent window appeared.

So, I'll release FluentWPF 0.4.0 soon.
Look forward to it :smile:
